mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -483,9 +551,16 @@ CLASS ail net/minecraft/entity/Entity
 	METHOD i setInvisible (Z)V
 		ARG 1 invisible
 	METHOD j stopRiding ()V
+	METHOD j teleport (DDD)V
+		ARG 1 destX
+		ARG 3 destY
+		ARG 5 destZ
 	METHOD j getCameraPosVec (F)Lcsa;
 		ARG 1 tickDelta
 	METHOD j allowsPermissionLevel (I)Z
+		ARG 1 permissionLevel
+	METHOD j getHardCollisionBox (Lail;)Lcrv;
```

Comment:
What's the "hard collision box"?

Reply:
It's used for hard collissions, as in boats and minecarts. Returning a non-null value here makes the entity effectively solid when interacting with the player and other entities, as opposed to the default where entities tend to squish together.